Premier league.ipynb

```diff
@@ -0,0 +1,1287 @@
+{
```

Comment:
<ol><li>Sort the teams by points explicitly.</li><li>Notice that top-5 is not well defined because the 6th place could have the same points as the 5th place. How do you propose to treat that fact ?</li><li>Use the <code>zip</code>  trick here too.</li></ol><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/A4Vision/games-course/pull/27/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>